tensilelite/Tensile/Components/LSU.py

```diff
@@ -298,6 +305,15 @@ def writeReadReduction(self, writer, kernel):
                     comment="lsu offset = lsu_id * LSU Process Offset"))
                 module.add(VAddU32(dst=vgpr(addr), src0=vgpr(addr), src1=vgpr(tmpVgpr), \
                     comment="addr += lsu offset"))
+                if  writer.states.archCaps["HasLDSGT64K"]:
+                    if maxOffset >= 65536*2:
```

Comment:
Same here.
We can use for loop instead of if statements.

for i in range(1, numAddr):
    offset = 0x10000 * i
    module.add(VAddU32(vgpr(addr+i),hex(offset), vgpr(addr), \
                            comment="addr += %d"%offset))